docs/developers/node-tutorial.mdx

```diff
@@ -1582,12 +1582,6 @@ Query node information:
 celestia rpc node Info
 ```
 
-Get share by data availibility header and index:
-
-```bash
-celestia rpc share GetShare "$(celestia rpc header GetByHeight 32236 | jq '.result.dah' -r)" 32236 0
-```
-
 Get share by namespace:
```

Comment:
[not blocking] but based on the issue, should `GetSharesByNamespace` also be removed?

Reply:
yes! thank you.